tools/crm_mon.c

```diff
@@ -1805,7 +1800,7 @@ mon_refresh_display(gpointer user_data)
     /* Unpack constraints if any section will need them
      * (tickets may be referenced in constraints but not granted yet,
      * and bans need negative location constraints) */
-    if (show & (mon_show_bans | mon_show_tickets)) {
+    if (is_set(show, mon_show_bans) || is_set(show, mon_show_tickets)) {
```

Comment:
As an aside, someday I'd like to revamp our bitwise functions, something like
pcmk_any_bits_set(mask, bits)
pcmk_all_bits_set(mask, bits)
pcmk_any_bits_unset(mask, bits)
pcmk_all_bits_unset(mask, bits)
so we don't have to look up the definition to know how multiple bits behave :)